### PR TITLE
Pull request for TSE Review

### PR DESCRIPTION
**Pull request by fdfragoso**
- All the features added as aked on pathwright and Odin project;
- Linters and HTML validator checked;
- Responsive added and tested with Chrome and Firefox;
- Waiting for feedback.

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @fdfragoso   & @JelilFaisalAbudu   :wave:
-
Good job so far! :tada: We have some few changes to do before we proceed (see review comment).

## [REQUIRED] - ADD - Missing list items.
In the nav-centre section, the growth quarters and neural list items are missing:
Heres yours: 👇 
nav_centre_clone
Heres the original: 👇 
nav_centre_section

## [REQUIRED] - REMOVE - The post date background colour.
- Change your post's date background colour to transparent. 

From this (your current): 👇 
post_date_clone

To this (to match the original): 👇 
post_date_original

## [REQUIRED] - CHANGE - The top content section to match the original.
Add some margin to your top three content
Form this (yours): 👇 
post_3_box_clone

To look like this (original): 👇 
post_3_box_original

## [REQUIRED] - CHANGE - TNW logo is not rightly placed.
Adjust the TNW logo's padding to match the original's site.

Form this: 👇 
tnw_clone

To this (original): 👇 
tnw_logo_original

## [REQUIRED] - ADD TNW Logo
When scrolling I did expect to see the TWN logo appearing at the leftmost part of your page.
Instead, I was met with this: 👇 
scroll_tnw_logo_clone

Expected to see: 👇 
scroll_tnw_logo

Kindly change this. It is a requirement 
scroll_tnw_logo_requirement

## [REQUIRED] -  CHANGE - The wrapper contents.
The wrapper needs to match the original.
This is what your [latest news section], [latest funding rounds on Index.co] and the rest of the content below. looks: ⬇️ 
below_3_section_clone

Change the size of your contents to match this: ⬇️ 
Below_top_3_sections

## [REQUIRED] - CHANGE- The Footer section.
Tweak your footer as well to match the original
From this: 🔽 
footer_clone

To look much like this: 🔽 
footer

## [REQUIRED] - Medium version changes
- The TWN logo should appear at the leftmost in a medium screen view.
This is what you have in medium view: 👇 
medium_screen_tnw_logo_clone

This is what is required: 👇 
medium_screen_tnw_logo


Please make carryon with changes and submit another review request.

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:

- When you have made the specified changes, please submit another code review. Thank you!

*Happy coding :smile:,*
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by fdfragoso** [comment]
@christianotieno please can you clear this error to us: "[REQUIRED] - CHANGE - The top content section to match the original." - would be a thin margin between the yellow bar and the content?

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [x] As the last TSE required 'When scrolling I did expect to see the TWN logo appearing at the leftmost part of your page.' the logo has to appear even when you scroll, consider follow the original page design.

- [x] As the last TSE required 'Adjust the TNW logo's padding to match the original's site.' please make the logo at the very left side of the page, follow the original page.

**Medium screen:**

- [x] As the last TSE required 'The TWN logo should appear at the leftmost in a medium screen view.' consider make the logo appears as the original site.

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by fdfragoso** [comment]
@SofiBretz - About your comments:

- The first error to solve it we need to use js. On the original web site, the logo shows up after scrolling a little bit down. We looked over the internet and everyone says to use JS.
- The second error we fixed on the first review. Our logo is on the left corner, maybe a few pixels to the right, but without any margin or padding. Comparing with the website we cant see the difference.
- It was already done. When in the middle screen size the logo on the navbar shows up.

**Comment by sumancrest0001** [comment]
[REQUIRED CHANGES] You guys have done a great job :clap: :clap: 
But we need to make a couple of changes to get a green signal.
[REQUIRED] 'When scrolling I did expect to see the TWN logo appearing at the leftmost part of your page.' I believe we can achieve the feature with CSS also. 

[Required] Please consider following images to make sure that it looks similar to the original page. For the medium and small screen.

Screenshot from 2020-04-02 15-22-21

It should be like this.
Screenshot from 2020-04-02 15-37-58

**Comment by sumancrest0001** [comment]
[Required] Please make 2 columns for the apps, gears section on the medium screen. Currently, its 3 columns. Do not forget to give some margins between columns. It will make your layout very good.
Screenshot from 2020-04-02 15-25-23

It should be like this :point_down:
Screenshot from 2020-04-02 15-25-48

**Comment by sumancrest0001** [comment]
[Required] you have to make changes to the section also. Please try to emulate the same design as that of the original site.
Screenshot from 2020-04-02 15-26-44

**It should be like this :point_down:** 
Screenshot from 2020-04-02 15-44-11
[TIPS] Try to reduce font-size, letter-spacing, and line-height on the medium screen.

Reach out to the slack channel if you need any support. **suman shrestha**

ALL THE BEST  :+1:

**Comment by sumancrest0001** [comment]
[CHANGES REQUIRED] Nice work :+1:  :+1:  but it seems like you forgot to fix the first issue stated in the previous review. 

As the last TSE required 'When scrolling I did expect to see the TWN logo appearing at the leftmost part of your page.' the logo has to appear even when you scroll, consider follow the original page design.

**You have fixed other issues. :clap:  :clap:  :clap:** 

But while doing so, Other issues are raised. Please check the following issues and submit once again after fixing those issue.
I know we are requesting you to make changes multiple times and it could be a little bit demotivating but trust me after fixing these issue, your project looks almost the same as that of the original site.

Stay motivated and kindly make the following changes.

**In your project**

Screenshot from 2020-04-02 19-03-01

### But it should look like this :point_down: 

Screenshot from 2020-04-02 19-03-11
[TIPS] remove justify-content: space-between property for the medium screen.

**Comment by sumancrest0001** [comment]
[REQUIRED CHANGES] We have to also change the font-size of the top articles. Please have a look and make changes. It won't take much time. 
Screenshot from 2020-04-02 19-07-18

### It should be :point_down:  :point_down: 
Screenshot from 2020-04-02 19-07-39

**Comment by sumancrest0001** [comment]
We are very close to getting :+1: :+1:  for this project.

Feel free to contact me on slack **suman shrestna**

### Happy coding

**Comment by fdfragoso** [comment]
@sumancrest0001 - Like we said before this would be done using javascript what is out of the scope for this project. If you have any suggestions on how to achieve it, great we will do it, but in the moment we can't do it.  As the last TSE required 'When scrolling I did expect to see the TWN logo appearing at the leftmost part of your page.' the logo has to appear even when you scroll, consider follow the original page design.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1183 @@
...
+
+/************************
+* NAVIGATION BAR STYLES *
+*************************/
+
+.nav {
+  width: 100%;
+  position: sticky;
+  top: -7%;
+  z-index: 99999;
```
Comment (men32z) [inline style.css:56]: change this before to merge, is causing sticky nav getting hide.
```suggestion
  top: 0;
  z-index: 99999;
```

**Review by men32z — APPROVED** [review]
## Good job making the changes,

before to merge take a look on this OPTIONAL changes that would make your project even better, 
- [ ] some images are distorted, you could fix them with ` background-size: cover;` for example but there are more ways to do it. 
- [ ] fix the sticky navbar.
- [ ] try to change some images to make it look more natural and less like a template.
- [ ] take a look on the original web site fonts styles, sizes and font family, that would make your project look more professional. 
- [ ] and last but not least, try to make the required change of the logo that my collegues ask for, remember that you can use google to find solutions [like this](https://stackoverflow.com/questions/21176131/check-scrolling-with-css), and yes you are right here the best approach is using JS which is out of the scope but somewhere in the curriculum you will see JS so I guess doesn't hurt to try to implement it

BlgS1sdIUAEybkW

### Status Approved

**Pull request merged**